python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/event-handlers.j2

```diff
@@ -8,6 +8,12 @@
 {%     for handler in event_handlers | arista.avd.natural_sort('name') %}
 !
 event-handler {{ handler.name }}
+{%         if handler.actions.log is arista.avd.defined(true) and not handler.actions.bash_command is  arista.avd.defined and handler.name != "trigger-on-boot" %}
```

Comment:
the logic should be on trigger not on handler name

Reply:
Fixed.